CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
 ## 0.0.1 (Unreleased)
```

Comment:
This changelog is not up to date. We're already at `0.1.6` and this would release in `0.1.7`
Maybe we should just get rid of this file as we manage changelog in the releases screen

Reply:
Made a separate PR for this - https://github.com/Twingate/terraform-provider-twingate/pull/88